src/typeahead/typeahead-container.component.ts

```diff
@@ -188,6 +188,9 @@ export class TypeaheadContainerComponent {
       e.preventDefault();
     }
     this.parent.changeModel(value);
+
+    value.selectEvent = e;
```

Comment:
I don't see usage of this field, plus it should be set to null on destroy.
Cuz event.target has link to DOM element which in turn could lead to memory leak via detached element persistance
In short: could be deleted?